cpp/ps0001/cpp_ps0001.cpp

```diff
@@ -1,26 +1,17 @@
-/* help your brother solve summation problems
-    155
-    940
-    +168
-    --------
-    1263
-*/
-
-
 #include <iostream>
 
 int main() {
-    int num1 , num2 , num3 , result;
-
+    // prompting user for 3 numbers to add 1/2
+    short num1 , num2 , num3 , result;
     std::cout << "Enter number 1" << std::endl;
     std::cin >> num1;
     std::cout << "Enter number 2" << std::endl;
     std::cin >> num2;
     std::cout << "Enter number 3" << std::endl;
     std::cin >> num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add